lib/license_finder/packages/erlangmk_package.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'rubygems'
+
+class InvalidErlangmkPackageError < ArgumentError
+end
+
+# NOTE:
+# Command:
+#   make QUERY='name fetch_method repo version absolute_path' query-deps
+# Output:
+# make[1]: Entering directory '/home/lbakken/development/rabbitmq/umbrella/deps/credentials_obfuscation'
+# make[1]: Leaving directory '/home/lbakken/development/rabbitmq/umbrella/deps/credentials_obfuscation'
+# rabbit_common: rabbitmq_codegen git_rmq https://github.com/rabbitmq/rabbitmq-codegen master /home/lbakken/development/rabbitmq/umbrella/deps/rabbitmq_codegen
+# rabbit_common: lager hex https://hex.pm/packages/lager 3.8.0 /home/lbakken/development/rabbitmq/umbrella/deps/lager
+# rabbit_common: jsx hex https://hex.pm/packages/jsx 2.9.0 /home/lbakken/development/rabbitmq/umbrella/deps/jsx
+# rabbit_common: ranch hex https://hex.pm/packages/ranch 1.7.1 /home/lbakken/development/rabbitmq/umbrella/deps/ranch
+# rabbit_common: recon hex https://hex.pm/packages/recon 2.5.1 /home/lbakken/development/rabbitmq/umbrella/deps/recon
+# rabbit_common: credentials_obfuscation hex https://hex.pm/packages/credentials_obfuscation 2.0.0 /home/lbakken/development/rabbitmq/umbrella/deps/credentials_obfuscation
+# lager: goldrush git https://github.com/DeadZen/goldrush.git 0.1.9 /home/lbakken/development/rabbitmq/umbrella/deps/goldrush
+
+module LicenseFinder
+  class ErlangmkPackage < Package
+    @word_re = Regexp.new('^\w+$')
+    @version_re = Regexp.new('\d+\.\d+\.\d+')
+    @version_prefix_re = Regexp.new('^v')
+
+    class << self
+      def new_from_show_dep(dep)
+        raise_if_not_valid(dep)
+        new(
```

Comment:
Correct me if I am missing something. But I don't see anything that fetches/denotes the license for any erLang packages. For example, we create each NuGet package with license for them (can be seen [here](https://github.com/pivotal/LicenseFinder/blob/master/lib/license_finder/package_managers/nuget.rb#L56)).

Reply:
I don't understand this comment. What does the output from `license_finder` look like when the `Nuget` package manager is used?

Here is the current (and expected) output when an `erlang.mk` project is queried:

```
$ bundle exec license_finder --project-path ./features/fixtures/erlangmk
LicenseFinder::Erlangmk: is active

Dependencies that need approval:
amqp_client, 3.8.5, unknown
aten, 0.5.3, "Apache 2.0"
cowboy, 2.6.1, ISC
cowlib, 2.7.0, ISC
credentials_obfuscation, 2.0.0, "Apache 2.0"
cuttlefish, 2.2.0, unknown
gen_batch_server, 0.8.2, "Apache 2.0"
getopt, 1.0.1, unknown
goldrush, 0.1.9, ISC
jsx, 2.9.0, MIT
lager, 3.7.0, "Apache 2.0"
lager, 3.8.0, "Apache 2.0"
observer_cli, 1.5.4, MIT
ra, 1.1.2, "Apache 2.0"
rabbit, 3.8.5, unknown
rabbit_common, 3.8.5, "MIT, New BSD"
rabbitmq_cli, 3.8.5, unknown
rabbitmq_codegen, 3.8.5, unknown
rabbitmq_management, 3.8.5, "Apache 2.0, ISC, MIT"
rabbitmq_management_agent, 3.8.5, unknown
rabbitmq_web_dispatch, 3.8.5, unknown
ranch, 1.7.1, ISC
recon, 2.5.1, "New BSD"
stdout_formatter, 0.2.2, "Apache 2.0"
syslog, 3.4.5, MIT
sysmon_handler, 1.2.0, "Apache 2.0"
```

`unknown` is due to the fact that some projects are still using MPL 1.1

Reply:
The output actually looks fine. Thanks!